composer.json

```diff
@@ -4,17 +4,17 @@
 
     "type": "library",
     "keywords": ["library", "aop", "php", "aspect"],
-    "homepage": "http://go.aopphp.com/",
+    "homepage": "https://go.aopphp.com/",
     "license": "MIT",
 
     "require": {
-        "php": "^7.4.0",
+        "php": "^8.0",
         "doctrine/annotations": "^1.11.1",
         "doctrine/cache": "^1.10",
         "goaop/parser-reflection": "^3.0.1",
```

Comment:
I don't see any support for PHP8 attributes here inside `goaop/parser-reflection` and this was main blocker for me to get PHP8 support, have you implemented support for PHP8 inside `goaop/parser-reflection`?

Reply:
I tried to remove the `goaop/parser-reflection` as you can see in my [fork-2 branch](https://github.com/WalterWoshid/goaop-framework/tree/fork-2), but somehow was not able to fully understand Better-Reflection to resolve parent classes (extends Class), because it always threw an Exception, that it couldn't find it in the [SourceLocator](https://github.com/Roave/BetterReflection/blob/6.5.x/docs/usage.md#sourcelocators)

See [getParent](https://github.com/WalterWoshid/goaop-framework/blob/7b3fe734a07f2f73177c451cc60a04213559296a/src/Core/Service/AdviceMatcher.php#L98) and [$parent->getProperties(...)](https://github.com/WalterWoshid/goaop-framework/blob/7b3fe734a07f2f73177c451cc60a04213559296a/src/Core/Service/AdviceMatcher.php#L170)

So I only upgraded the parameter and return types to support PHP 8 and PHP 8.1

Reply:
And those changes don't have backwards compatibility for older versions. I think supporting the [currently supported PHP version](https://www.php.net/supported-versions.php) is more important than providing support for all versions, which in my opinion I would want to, but it's too difficult to focus on multiple versions

Reply:
Depends on https://github.com/goaop/parser-reflection/pull/122

Reply:
I think we can create a new major versions based on your work in both projects. Then we will have an opportunity to add `branch-alias` and RC/beta tags to test them together and put a requirement on both of them to work with PHP>=8.0. Keeping BC with version 7.4 is not required at all.